RELEASE_PROCESS.md

```diff
@@ -28,8 +28,10 @@ In order to release, you will need the following accounts/permissions:
 account.)
 - Access to artifactory.chef.co
 - Access to delivery.chef.co
-- The CHANGELOG_GITHUB_TOKEN environment variable set to a github token gathered here: https://github.com/settings/tokens/new?description=GitHub%20Changelog%20Generator%20token
-- Install Github Changelog Generator: `gem install github_changelog_generator`
+- The CHANGELOG_GITHUB_TOKEN environment variable set to a github token gathered
+  [here](https://github.com/settings/tokens/new?description=GitHub%20Changelog%20Generator%20token).
```

Comment:
What permissions do I need to give it?  So far, I gave it read access to repos/commits - I haven't gone far enough through the checklist to know if that's enough.